mozilla-lockwise/lockwise-ios
# Disable dark mode UI overrides in Credential Provider extension

Fixes #1182 

When device is set to dark mode and attempts to autofill a credential by viewing the list in Lockwise, Dark Mode UI defaults were turning the domain font to an unreadable color, and it appeared as if the domain name was not being displayed.  

Dark Mode UI was already turned off in the Lockwise info.plist, but not in the Credential Provider extension info.plist. This change handles that.

## Testing and Review Notes

- Set iOS device to dark mode
- When prompted to autofill a password, click the option to open the list in Lockwise
- verify domain names are visible above username/email data on the list
